R/httpgd.R

```diff
@@ -347,16 +335,15 @@ hgd_svg <-
 #' @export
 #'
 #' @examples
-#' \dontrun{
-#'
 #' hgd()
+#'
 #' plot(1, 1)
-#' s <- hgd_svg(width = 600, height = 400)
+#' s <- hgd_plot(width = 600, height = 400)
+#'
 #' hist(rnorm(100))
-#' hgd_svg(file = tempfile(), width = 600, height = 400)
+#' hgd_plot(file = tempfile(), width = 600, height = 400)
```

Comment:
I don't think we should run code that creates files automatically ever. so maybe this could be wrapped in `if (interactive())` as well? (same thing above)

Reply:
examples of using tmpfile exists.
https://github.com/apache/arrow/blob/295c6644ca6b67c95a662410b2c7faea0920c989/r/R/csv.R#L698-L701
(I added a process similar to arrow because it might be better to remove these)

CRAN seemed to be concerned that examples were not checked, so I think we can leave it as is.

Reply:
Thanks! Deleting them does seem better.